kratos/includes/element.h

```diff
@@ -689,17 +689,17 @@ class Element : public GeometricalObject, public Flags
      * @param rDestinationVariable variable in the database to which the rRHSVector will be assembled
       * @param rCurrentProcessInfo the current process info instance
      */
-    virtual void AddExplicitContribution(const VectorType& rRHSVector, const Variable<VectorType>& rRHSVariable, Variable<double >& rDestinationVariable, const ProcessInfo& rCurrentProcessInfo)
+    virtual void AddExplicitContribution(const VectorType& rRHSVector, const Variable<VectorType>& rRHSVariable, const Variable<double >& rDestinationVariable, const ProcessInfo& rCurrentProcessInfo)
```

Comment:
@RiccardoRossi I added also the const here to the variable, since you spotted it in #3715